src/app/wallets/get-balance-for-wallet.ts

```diff
@@ -12,18 +11,11 @@ export const getBalanceForWallet = async ({
   lock?: DistributedLock
   logger: Logger
 }): Promise<Satoshis | ApplicationError> => {
-  const [, updatePaymentsResult] = await Promise.all([
-    updatePendingInvoicesByWalletId({
```

Comment:
Why not `declineHeldInvoices` at this point as well?

I noticed we decline in the `checkIsBalanced` test helper.